kata-containers-image/debian.control

```diff
@@ -0,0 +1,12 @@
+Source: kata-containers-image
+Section: devel
+Priority: optional
+Maintainer: Kata containers team <https://github.com/kata-containers/>
+Standards-Version: 3.9.6
+Homepage: https://katacontainers.io
+Build-Depends: debhelper (>= 9), flex, bison
+
+Package: kata-containers-image
+Architecture: amd64
```

Comment:
Hard-coded arch. It might be best to rename this file `debian.control.in` and build it using `m4` or `sed` so that the `Architecture` field can be set to whatever `$(arch)` is. Same comment to a few other files below.